flink-metrics/flink-metrics-dropwizard/src/main/java/org/apache/flink/dropwizard/ScheduledDropwizardReporter.java

```diff
@@ -74,6 +75,15 @@ protected ScheduledDropwizardReporter() {
 	}
 
 	// ------------------------------------------------------------------------
+	//  Getters
+	// ------------------------------------------------------------------------
+
+	// used for testing purposes
+	Map<Counter, String> getCounters() {
```

Comment:
You could also access the protected registry and get the counters from there.

We may not even need the gauges/counters/histograms fields in the ScheduledDropwizardReporter.